.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,68 @@
+name: Publish Docker image
+
+on:
+  push:
+    tags:
+      - "arize-phoenix-v*"
```

Comment:
It would also be helpful to publish an image from the tip of `main`.

Reply:
we'd probably want a separate job for that then?

Reply:
Not sure. This is non-blocking for this PR, but a nice-to-have.

Reply:
If we tackle this, we should also tag by the commit hash.